_posts/2018-07-17-element-existentialism.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: Element Existentialism
+date: 2018-07-17
+author: Titus Fortner
+author_url: http://watirtight.com/
+comments: true
+categories:
+---
+
+To Be, or Not to Be... an Element
+<!--more-->
+
+### TL/DR
+
+`Element#visible?` will soon do the same thing as `Element#present?`, which has the potential to cause problems if
+you rescue `UnknownObjectException` or `UnknownFrameException` in your code. While we aren't officially 
+deprecating `#visible?`, we recommend that you make sure that you need to be using any existence check in the first place 
+(see point #1 in the next section), and if you do, switch to using `#present?` instead.
+
+### What Matters
+
+There are a bunch of different terms that mean slightly different things when it comes to whether an element "exists".
+See the [Terminology section](#terminology) below for the meaning of the terms used in quotes in this article.
+
+There are two major uses cases for evaluating whether an element is. 
```

Comment:
s/uses/use

Reply:
End should also be "exists" instead of "is".